docs/running.rst

```diff
@@ -126,19 +128,25 @@ Alternatively, to run the simulation in MPI mode:
 
 .. code-block:: none
 
-   $ yank script --yaml=yank.yaml
+   $ mpirun -n # yank script --yaml=yank.yaml
+
+where ``#`` is the number of parallel instances of YANK you want to run with. Keep in mind your system may have a
+different MPI executable name and/or syntax.
 
 On systems with multiple NVIDIA GPUs per node, it is necessary to perform masking using ``CUDA_VISIBLE_DEVICES``.
 
 On systems using the conda-installed ``mpi4py`` package, the `MPICH2 hydra mpirun <https://wiki.mpich.org/mpich/index.php/Using_the_Hydra_Process_Manager>`_ will be automatically installed for you.
-You can use the cluster utility script `build-mpirun-configfile.py <https://github.com/choderalab/clusterutils/blob/master/scripts/build-mpirun-configfile.py>`_ available in our `clusterutils <https://github.com/choderalab/clusterutils>`_ tools to generate an appropriate ``configfile``:
+You can use the cluster utility script
+`build-mpirun-configfile.py <https://github.com/choderalab/clusterutils/blob/master/clusterutils/build_mpirun_configfile.py>`_
```

Comment:
We don't use the Python executable directly---we use the conda-installable `clusterutils` package that exposes the command-line executable `build-mpirun-configfile`. I don't think we want to confuse people by specifying both the Python script and command-line form.